lib/handlers/PostHandler.js

```diff
@@ -15,6 +15,10 @@ class PostHandler extends BaseHandler {
      * @return {function}
      */
     send(req, res) {
+        if ('upload-concat' in req.headers && !this.store.hasExtension('concatentation')) {
+            return Promise.resolve(super.send(res, 400, {}, 'Concatenation extension is not supported'));
```

Comment:
I think the extra hint on how to fix it is helpful
```suggestion
            return Promise.resolve(super.send(res, 400, {}, 'Concatenation extension is not (yet) supported. Disable parallel uploads in the tus client. '));
```